simcal/jwst_tests/imaging/test_nircam.py

```diff
@@ -6,6 +6,15 @@
 from jwst.pipeline import Detector1Pipeline
 from jwst.pipeline import Image2Pipeline
 
+from ci_watson.artifactory_helpers  import (check_url, get_bigdata_root,
+    get_bigdata,
+    BigdataError,
+)
+
+
+data_root = get_bigdata_root()
+
+import shutil
```

Comment:
We're also no longer copying files into a subdir, so no need for this import either.